db/organizations/viafoura.eno

```diff
@@ -1,11 +1,9 @@
 name: Viafoura
 website_url: http://www.viafoura.com/
 privacy_policy_url: https://viafoura.com/privacy-policy/
-privacy_contact: privacy@viafoura.com
-country: 
-description: "Viafoura’s mission is to elevate the quality and monetize conversations on the internet.
-| 
-| Our technology has been designed to accomplish this task for large online publishers. Our easy to install plugins enable social login, commenting, content discovery and gamification as a way to gain deeper user insight. This data increases monetization opportunities in a variety of ways such as increased ad revenues and better informed content creation that keeps users engaged."
+privacy_contact: privacyofficer@viafoura.com
```

Comment:
Based on the policy, looks like the old value privacy@viafoura.com is still correct. I also did not see "privacyofficer@viafoura.com" being listed.